packages/components/tag/src/tag.visualroute.jsx

```diff
@@ -199,9 +199,7 @@ export const component = () => (
         {longText}
       </Tag>
     </Spec>
-    <Spec
-      label="Tag list - with remove"
-    >
+    <Spec label="Tag list - with remove">
       <TagList>
         {examplesLong.map((tag, index) => (
           <Tag type="normal" to="foo/bar" onRemove={() => {}} key={index}>
```

Comment:
Does it make sense to update these tests to use `tone` instead of `type`?

Reply:
I didn't change existing tests, so that I can make sure the existing `type` property continues to work.

Reply:
fair enough, im fine with updating when we remove the prop